tickets/app.py

```diff
@@ -0,0 +1,140 @@
+import os
+
+from datetime import datetime
+
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from flask import request
+from config import TICKET_STATUS_OPEN, TICKET_STATUS_CLOSED, TICKET_STATUS_ANSWERED, TICKET_STATUS_WAITING_FOR_ANSWER, \
+    TICKET_STATUSES, REDIS_TICKET_COMMENTS_KEY_TEMPLATE
+import json
+import redis
+import uuid
+
+
+app = Flask(__name__)
+env_config = os.getenv("APP_SETTINGS", "config.DevelopmentConfig")
+app.config.from_object(env_config)
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+migrate = Migrate(app, db)
+r = redis.Redis(db=1)
+
+
+class Ticket(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    subject = db.Column(db.String(256), nullable=False)
+    text = db.Column(db.Text, nullable=False)
+    email = db.Column(db.String(256), nullable=False)
+    status = db.Column(db.String(16), nullable=False)
```

Comment:
Использовать Status enum и дефолтное значение open
```suggestion
    status = db.Column(db.Enum(Status), nullable=False, default=Status.OPEN.value)
```